test/libsolidity/lsp/references/import_aliases.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "../goto/lib.sol" as Imported;
+//                          ^ @ImportedDef
```

Comment:
Please also add a test where the cursor is outside of any definition. Or inside a comment. Or pointing at a keyword. Or at a literal. Or at a word inside a string.

Reply:
I'm just trying to make sure that the LSP will behave in a sensible way in these cases. They're also a part of the feature. I don't really mean to test AST locations themselves. The fact that they're coming straight from the AST is an implementation detail. The LSP could be doing some transformation on them in some cases if necessary. Or not. I see it as a black-box test of this feature's behavior.